.github/workflows/post_summaries.yaml

```diff
@@ -0,0 +1,64 @@
+name: Comment on the pull request
```

Comment:
Can you add some (brief) line comments describing what this workflow does / is for?